coreneuron/utils/randoms/nrnran123.cu

```diff
@@ -10,10 +10,13 @@
 #include "coreneuron/utils/nrnmutdec.hpp"
 #include "coreneuron/utils/randoms/nrnran123.h"
 
+#include <boost/pool/pool_alloc.hpp>
+
```

Comment:
⚠️ 🔴

Reply:
With the latest commit it is optional -- GPU builds on systems without Boost will "just" be slow (and awkward to profile the initialisation of).